ONSdigital/ssdc-rm-ddl
# 464 adding indexes into ddl model

# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

  version: 

# Motivation and Context
New indexes and run the indexes on the 

# What has changed
added new patch
new file for running in GIN indexes, now run on Groundzero

# How to test?
Check patch 1st
Then run in the ddl groundzero in the env of your choice.  Has it created the indexes expected?

# Links
https://trello.com/b/VS0ZyRmj/srm
